modin/tests/core/storage_formats/pandas/test_internals.py

```diff
@@ -2784,3 +2784,23 @@ def dataframe_test_default_property(df):
         match="<function DataFrame.<property fget:_test_default_property>> is not currently supported",
     ):
         pd.DataFrame([[1]]).dataframe_test_default_property
+
+def test_daemonic_worker_protection():
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "modify_config",
    [
        {Engine: "Ray"},
        {Engine: "Dask"},
    ],
    indirect=True,
)
def test_daemonic_worker_protection():
```
We test modin/tests/core/storage_formats/pandas/test_internals.py only with Ray engine in CI. We should either extend CI matrix to run on different engines or modify the engine directly in the test. I propose to go with the second option for now.